server/manifest.go

```diff
@@ -0,0 +1,75 @@
+package server
+
+import (
+	"bytes"
+	"crypto/sha256"
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+
+	"github.com/ollama/ollama/types/model"
+)
+
+type Manifest struct {
+	ManifestV2
```

Comment:
Why leave ManifestV2 in `images.go`?

Reply:
I didn't want to make changes to ManifestV2 yet. That'll come in a later change